normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
```

Comment:
Please capitalize ASAP. Also, please change to "Issue must be fixed ASAP", it isn't always related to production breaking.

Reply:
Updated in caf892e.